### PR TITLE
Redesigned navbar

### PR DESCRIPTION
**Pull request by jacemonje**
One part of many for https://github.com/codetriage/codetriage/pull/352/ 

This deals with the rearranged navbar and the addition of a "repos" link

screen shot 2015-06-25 at 7 31 52 am

### CODE REVIEW

File: app/views/application/_nav.html.slim [hunk 1]
```diff
@@ -11,3 +12,5 @@
         = link_to 'Sign in', new_user_registration_path
   h3.muted
     = link_to "CodeTriage", root_path
+
+    / = render :partial => 'repos/search'
```
Comment (prathamesh-sonpatki) [inline app/views/application/_nav.html.slim]: Should this be removed? I guess it's leftover.

File: db/schema.rb [hunk 2]
```diff
@@ -16,10 +16,10 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
```
Comment (prathamesh-sonpatki) [inline db/schema.rb]: I have opend another PR for tracking changes to schema.rb. Can you please remove these changes from this PR?

**Comment by jacemonje** [comment]
Removed them!

**Comment by prathamesh-sonpatki** [comment]
@jacemonje Please rebase with latest master.

**Pull request closed without merging**